examples/neighbors/plot_lof.py

```diff
@@ -0,0 +1,51 @@
+"""
+==========================================
+LOF example
+==========================================
+
+An example using LOF for anomaly detection.
```

Comment:
You should be more explicit.

LOF is jargon. I vote to rename the object LocalOutlierFactor to be explicit.
Also never use the acronym without writing the full words.

you could add a ref to the doc of this example too.